extensions/src/AWSSDK.Extensions.Bedrock.MEAI/BedrockChatClient.cs

```diff
@@ -704,20 +705,30 @@ private static Document ToDocument(JsonElement json)
                 }
             }
 
+            var schemaDictionary = new Dictionary<string, Document>()
+            {
+                ["type"] = new Document("object"),
+            };
+
+            if (inputs != default)
+            {
+                schemaDictionary["properties"] = inputs;
+            }
+
+            if (required.Count > 0)
+            {
+                schemaDictionary["required"] = new Document(required);
+            }
```

Comment:
This is the crux of the change, not setting these into the schema when they're empty? It's interesting to me that causes a service failure, that it makes a distinction between "properties" not being there and "properties" being there but with an empty array. But if it fixes an issue, great.